bin/docsify

```diff
@@ -1,124 +1,126 @@
 #!/usr/bin/env node
 
-'use strict'
+"use strict";
```

Comment:
This commit contains much more changes than removing gulp.
So it becomes much more harder to review this when other content especially just formatting is mixed.
Maybe we add some information about this to a CONTRIBUTION.md?

Reply:
Can you please give some feedback before resolving this conversation?

Reply:
Sorry, I didn't notice the mail.

Because I changed the configuration rules of eslint, it is consistent with the docsify project. This commit does not add or remove any features. So I am not strictly divided into multiple commits.

Reply:
Sure it didn't affect any user API, but still this would be impossible to revert for instance. I would expect to have one commit which removes gulp and another which would adapt the formatting via eslint.
Does it make sense to you?

Reply:
Sorry, I have merged PR 😂